mslice/models/cut/cut_cache.py

```diff
@@ -0,0 +1,78 @@
+
+class CutCache(object):
```

Comment:
Good call to create a type for this. How about just `Cut` for the type name? I would naively expect a cache to generally be a list of old cuts - this essentially encapsulates a cut as far as I can see?

Reply:
I agree. I'll change the slice equivalent as well.